From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Tue, 9 Apr 2019 08:22:13 +0200
Subject: [PATCH] don't build with -Werror

New warning may show up with different compiler versions.
Also, building with icecc may result in errors like this:

.../lib/v4l2.c:180:23: error: this statement may fall through [-Werror=implicit-fallthrough=]

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2cb8c20d9f23..4b0e1106f299 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.0)
 project(uvc-gadget)
 set(CMAKE_BUILD_TYPE Release)
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Wno-unused-parameter -Werror")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Wno-unused-parameter")
 
 include_directories(${CMAKE_BINARY_DIR}/generated)
 
